src/components/common/FilterGroup/useUrlFilters.ts

```diff
@@ -10,10 +10,10 @@ import type { GlobalFilters } from './types';
 
 /**
  * Safely parses a JSON string.
- * @param {string} jsonString - The JSON string to parse.
+ * @param {string} str - The JSON string to parse.
  * @returns {any} - The parsed JSON object or null if parsing fails.
  */
-const safeParse = (str: string) => {
+const safeParse = (str: string): any => {
```

Comment:
why using any and not unknown here? I also wonder why the eslint didn't caught with this any